### ISSUE TITLE
deploy command

### ISSUE THREAD

**Issue by jairhenrique** [luizalabs/teresa#293]
It would be nice if the command `teresa deploy` ask to me what environment I want to run the command.

**Comment by yagonobre** [comment]
You can use the `--config` flag to specify the environment

**Comment by jairhenrique** [comment]
@yagonobre `--config` doesn't work. I need to switch the environment with `config env-set`.

**Comment by yagonobre** [comment]
Yes, you are right, we need a `--context` :)

**Comment by jairhenrique** [comment]
Each developer can create `N` environments.

```
Deploying app "my-project" to the cluster "staging"...
Are you sure? (yes/NO)? NO

Do you want to deploy in one of these environments?
# LIST OF ENVIRONMENTS ON CONFIG
(env-name/NO)? production
```

@yagonobre what do you thing about this? ☝️

**Comment by drgarcia1986** [comment]
@jairhenrique be careful with interactive commands. 
Interactive commands don't work on CI process.

**Comment by yagonobre** [comment]
Check this #308

**Comment by yagonobre** [comment]
closed by #308, now you can use `--cluster` to select what cluster you will use
